### PR TITLE
add new specs

### PR DESCRIPTION
**Pull request by imahungrypanda**

### CODE REVIEW

File: spec/numerical_spec.rb [hunk 1]
```diff
@@ -62,3 +62,67 @@
     factorial(6)
   end
 end
+
+describe "#fibonacci" do
+  it "returns zeroith fibonacci number" do
```
Comment (iserbit) [inline spec/numerical_spec.rb]: should say "returns 0 when no number is requested."  actually, does this even need to be tested?

File: spec/numerical_spec.rb [hunk 2]
```diff
@@ -62,3 +62,67 @@
...
+    expect(fibonacci(0)).to eq(0)
+  end
+
+  it "returns first fibonacci number" do
+    expect(fibonacci(1)).to eq(1)
+  end
+
+  it "returns first two fibonacci numbers" do
```
Comment (iserbit) [inline spec/numerical_spec.rb]: should be "returns the second fibinacci number"

File: spec/numerical_spec.rb [hunk 3]
```diff
@@ -62,3 +62,67 @@
...
+    expect(fibonacci(2)).to eq(1) # = [0!, 1!]
```
Comment (iserbit) [inline spec/numerical_spec.rb]: i think the # =[0!, 1!] section is confusing.  you are asking for the value of the second number, not an array with two elements.

Comment (imahungrypanda) [inline spec/numerical_spec.rb]: Yeah. Copy and paste error

**Pull request merged**